File: src/Projects/Android/012-ImageProcessingServerNetLib/Server/src/main/java/org/csystem/app/imageprocessing/server/configuration/datetime/formatter/DateTimeFormatterConfig.java
package org.csystem.app.imageprocessing.server.configuration.datetime.formatter;

import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

import java.time.format.DateTimeFormatter;

@Configuration
public class DateTimeFormatterConfig {
    @Bean
    public DateTimeFormatter dateTimeFormatter()
    {
        return DateTimeFormatter.ofPattern("yyyy-MM-dd-HH-mm-ss-n");
    }
}

File: src/Projects/Android/013-RandomTextGeneratorServer/Server/src/main/kotlin/org/csystem/app/random/text/server/configuration/datetime/DateTimeFormatterConfig.kt
package org.csystem.app.random.text.server.configuration.datetime

import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration
import java.time.format.DateTimeFormatter

@Configuration
class DateTimeFormatterConfig {
    @Bean
    fun dateTimeFormatterText(): DateTimeFormatter = DateTimeFormatter.ofPattern("dd-MM-yyyy-HH-mm-ss-n");
}